iodata/test/test_orcalog.py

```diff
@@ -44,7 +44,10 @@ def test_load_water_number():
         mol.atcoords[0] - mol.atcoords[2]) / angstrom, 0.9500, atol=1.e-5)
     assert_allclose(np.linalg.norm(
         mol.atcoords[1] - mol.atcoords[2]) / angstrom, 1.5513, atol=1.e-4)
+    # check energies of scf cycles
+    energies = np.array([-76.34739931, -76.34740001, -76.34740005, -76.34740029])
+    assert_allclose(mol.extra['scf_energies'], energies)
     # check scf energy
-    assert_allclose(mol.energy, -74.959292304818, atol=1e-8)
+    assert_allclose(mol.energy, -76.347791524303, atol=1e-8)
```

Comment:
The energy values (total/SCF) are parsed properly, but a naive question about the ORCA energy values. I am confused about why the total energy doesn't match the energy of the last iteration. Checking the log file, the total energy corresponds to the 4th SCF cycle.

Reply:
I think the reason is that ORCA uses a finer grid for the final evaluation of the energy than for the SCF iterations.

Reply:
It could also be for technical reasons, e.g. one may compute a cheaper approximation of the energy which is sufficient for just checking SCF convergence. E.g. the sum of occupied orbitals energies plus the usual correction can be cheaper but is not exact when the orbitals have not fully converged.